fhir-server/liberty-config/configDropins/overrides/datasource-postgresql.xml

```diff
@@ -0,0 +1,18 @@
+<server>
+    <!-- ============================================================== -->
+    <!-- TENANT: default; DSID: default; TYPE: read-write               -->
+    <!-- ============================================================== -->
+    <dataSource id="fhirDatasourceDefaultDefault" jndiName="jdbc/fhir_default_default" type="javax.sql.XADataSource" statementCacheSize="200" syncQueryTimeoutWithTransactionTimeout="true">
```

Comment:
Shouldn't these ids be unique across all overrides? 
I think this should go in the disabled directory and note the overrides.

Reply:
This datasource gets used for Postgres tests where the tenant/dsid is default/default. The test automation script copies the correct fhir-server-config file to the target volume, and now also copies the relevant datasource overrides. I agree if we want to support tests with multiple tenants/dsids active at the same time that the id and jndiName values need to be unique. Although I provided a default implementation for calculating the JNDI names based on tenant/datasource, in some circumstances it's probably better (for readability) to include the jndiName property in the fhir-server-config so that the reference is clear. I made a similar comment in the doc.